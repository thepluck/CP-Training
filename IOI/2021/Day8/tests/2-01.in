12 15
3 2
8 7
3 4
11 3
4 10
7 1
1 2
12 3
6 5
5 1
3 9
10
1 7 4
1 3 5
1 11 5
2 3
1 9 1
1 10 2
2 7
1 6 2
1 5 3
2 9
