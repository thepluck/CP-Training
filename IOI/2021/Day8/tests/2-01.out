7726160
839800
105840
420
15015
3960
420
6435
792
210
792
